typescript/sdk/src/middleware/liquidity-layer/LiquidityLayerRouterDeployer.ts

```diff
@@ -64,30 +66,38 @@ export class LiquidityLayerDeployer extends MiddlewareRouterDeployer<
     super(multiProvider, configMap, liquidityLayerFactories, create2salt);
   }
 
+  routerContractName(): string {
+    return 'liquidityLayerRouter';
+  }
+
+  router(contracts: LiquidityLayerContracts): LiquidityLayerRouter {
+    return contracts.liquidityLayerRouter.contract;
+  }
+
   async enrollRemoteRouters(
     contractsMap: ChainMap<LiquidityLayerContracts>,
   ): Promise<void> {
     this.logger(`Enroll LiquidityLayerRouters with each other`);
     await super.enrollRemoteRouters(contractsMap);
 
     this.logger(`Enroll CircleBridgeAdapters with each other`);
+    // Hack to allow use of super.enrollRemoteRouters
     await super.enrollRemoteRouters(
```

Comment:
why are we doing this twice?

Reply:
The LiquidityLayerRouter, CircleBridgeAdapter, and PortalAdapter, are all router contracts that need enrollment

Reply:
oh...